demo-default.env

```diff
@@ -52,22 +52,36 @@ IGL_ALLOW_UNSAFE_REPO_IS_EMPTY=True
 IGL_ALLOW_UNSAFE_REPO_CLEAR=True
 
 
-IGL_MCHR_ROUTING_TABLE=[{ "Name": "shared db channel to Australia", "Jurisdiction": "AU", "Predicate": "UN.CEFACT.", "ChannelUrl": "https://sharedchannel.services.devnet.trustbridge.io/", "ChannelAuth": "Cognito/JWT" }, { "Name": "shared db channel to Singapore", "Jurisdiction": "SG", "Predicate": "UN.CEFACT.", "ChannelUrl": "https://sharedchannel.services.devnet.trustbridge.io/", "ChannelAuth": "Cognito/JWT" }, { "Name": "Local FR channel", "Jurisdiction": "FR", "Predicate": "UN.CEFACT.", "ChannelUrl": "http://docker-host:7500/", "ChannelAuth": "None" }]
-
-# this env variable contains the next JSON (just newlines are stripped):
+# This env variable...
+IGL_MCHR_ROUTING_TABLE=[{"Name":"shared db channel to Australia","Jurisdiction":"AU","Predicate":"UN.CEFACT.","ChannelUrl":"https://sharedchannel.services.devnet.trustbridge.io/","ChannelAuth":"Cognito/JWT","ChannelAuthDetails":{"client_id":"XX","client_secret":"YY","token_endpoint":"https://xx/","scope":"https://sharedchannel.services.devnet.trustbridge.io/full"}},{"Name":"shared db channel to Singapore","Jurisdiction":"SG","Predicate":"UN.CEFACT.","ChannelUrl":"https://sharedchannel.services.devnet.trustbridge.io/","ChannelAuth":"Cognito/JWT","ChannelAuthDetails":{"client_id":"XX","client_secret":"YY","token_endpoint":"https://xx/","scope":"https://sharedchannel.services.devnet.trustbridge.io/full"}},{"Name":"Local FR channel","Jurisdiction":"FR","Predicate":"UN.CEFACT.","ChannelUrl":"http://docker-host:7500/","ChannelAuth":"None"}]
```

Comment:
is ENV var the only way to configure the parameters? Maybe we can consider to such arrays of objects via config files (yaml, json)?

Reply:
it's the simplest way I think - given we need to provide it to the docker containers for cloud deployment and for docker containers locally + tests. It's uncomfortable to update but such updates are rare.

Reply:
I originally suggested hard coding it in Python until we had stuff working, then to build a more permanent solution using DynamoDB or some other lightweight db. For dev purposes, hardcoding is enough to get it working, but ultimately we don't want to have it as an env var as we will need to change it on the fly.

Reply:
I advise not to spend time improving things like that while we have other things not working at all :-)

Reply:
Agreed, that's why I said hard code it as a python dict - if an env var is causing pain, go back to the dict in code. But now isn't the time to implement the lightweight db solution. I think we are all on the same page :-)